src/query.js

```diff
@@ -0,0 +1,29 @@
+export const testCoverage = (revision, fileName) => {
+  return {
+    "from":"coverage",
+    "where":{"and":[
+      {"eq":{"source.file.name":`${fileName}`}},
+      {"eq":{"repo.changeset.id12":`${revision}`}}
+    ]},
+    "limit":1000,
+    "format":"table"
```

Comment:
The "table" format will return a `data` which is a list of tuples.  I do not know if that is easiest to work with.  `"format":"list"` will return a list of objects; which is sometimes easier because you can access by name rather than by column number.

Reply:
Cool, I'll try that out.